python/dgl/graphbolt/subgraph_sampler.py

```diff
@@ -191,6 +210,19 @@ def _node_pairs_preprocess(self, minibatch):
             compacted_negative_dsts if has_neg_dst else None,
         )
 
+    def _sample(self, minibatch):
+        (
+            minibatch.input_nodes,
+            minibatch.sampled_subgraphs,
+        ) = self.sample_subgraphs(
+            minibatch._seed_nodes, minibatch._seeds_timestamp
+        )
+        return minibatch
+
+    def sampling_stages(self, datapipe):
+        """The sampling stages are defined here by chaining to the datapipe."""
```

Comment:
I think this API is added for offering CustomizedSampler a chance to re-org the datapipe. As it's already exposed in this top level class, we'd better talk more about this API in case some advanced user may want to utilize it?

Reply:
I mentioned about overriding either sample_subgraphs or the sampling_stages method in the main docstring of SubgraphSampler.